### ISSUE TITLE
SUGGESTION for a MINI-MOD - 'Redstone Tool & Armour Set'

### ISSUE THREAD

**Issue by markygnlg** [canitzp/SolarHelmet#3]
**TL:DR: OP redstone armour & tools mini-mod**
- Hopefully for 1.12 to any current version!! (one mod for multiple versions rather than multiple mods by multiple authors - ie. wool2string, RottFlesh2Leather, WaterInNether, Lava'Pond'Regen, Cobweb2String, etc - soooo many mods by soooo many authors...)

 The reason why I ask for a redstone mini-mod recipe is that we'll need: -> wood pickaxe -> stone pickaxe -> iron pickaxe in order to get down to the deeper redstone levels

I'm hoping for an OP redstone tool & armour set such as:
[Redstone-Tools](https://www.curseforge.com/minecraft/mc-mods/redstone-mod-by-poetic_d) [v1.15 & v1.16.1 - but works on later versions if you know how to update the mods.toml file];
or [Redstone Gear Mod](https://www.curseforge.com/minecraft/mc-mods/redstone-gear-mod-1-15-2) [v1.14 but also works on v1.15]

... . Maybe a 1.12 version too as sooooo many OP tools &/or armour mods for 1.12 have waaaaaaaaaaaayyyyyyyyyyyyy toooo much junk (but I can't remember if 1.12 has redstone).

ATM, I need to edit the mods.toml file in Redstone-Tools to get it to work on 1.16 newer versions, I also discovered that the Redstone Gear Mod for v1.14 worked on v1.15  (I posted that on the modpage & the mod's author changed the description to make it a 1.15 version, since no one seems to test mods - ppl just immediately demand for **"an-updated-version-NOW"** or demand from the author to **"make-a-version-for-me-for-the-version-I'm-currently-playing"** - - as I'm sure you know!

...

**Comment by markygnlg** [comment]
When I see requests similar to this, can I give them your Discord? 
....Yes I know it's 'ALL CAPS' & double posted ..but if it might be a mini-mod ...
![delme-gd guy](https://user-images.githubusercontent.com/62978705/127745350-db1f3540-9e18-497c-a324-d3a83a9eaf90.png)

... Or your SuggestionBox Discord (or any other social media - as MrCrayfish has with his Discord & mrcrayfish.com)

**Comment by canitzp** [comment]
Can you please write the exact features of your idea down? It seems to me to be a bit too much for a mini-mod.

For the second post, well the problem is that my discord inbox could get filled with spam. They can come to me and for example write a curseforge pm or like you open an issue somewhere. But I have to say that I don't really want to continue "old" mods. It's fine for me, if the mod theme is what I like, but even then it normally is a lot of work.

**Comment by markygnlg** [comment]
I'm Sorry my friend, I misunderstood - I thought you were looking for any new ideas for mini-mods since your job change.

What I was trying suggest is an 'overall modding principle' that I've been dreaming of for years, ie.:

1.) I'd find my favourite mods that I use regularly for every/any version of a game & help other authors update/create/take-over those mods - & then try to make them work for as many versions as my skills would allow;

2.) my 2nd dream is similar to the 1st - it is to find popular 'outdated / unsupported' mods that I DON'T use & try to help the authors either update it or give me permission to do so (via author's permission or github fork).

3.) my 3rd dream is to create/modify/update any (popular) mini-mod & make it available for any popular version of the game (ie. mc1.09, 1.12, 1.14, 1.16, etc) - so that everyone can come to one place for a specific mod instead of using multiple search engines to find each mod for each game version (eg. minecraft's flesh2leather, wool2string, cobweb2string, old lava regen, redstone pwr levels, lower shield, etc.).


I only ask you because MCreator is suggested to be an easy way to mod - I wanted to do these ideas myself via MCreator - but it is not available for current versions.

Thanks for your time,
Take care & all the best to you.
